sb.web.app/src/components/SaldoCard.vue

```diff
@@ -20,7 +20,7 @@
             <a href="#">
               <img src="../assets/sidecard_logos/credit-card.png"/>
               <!-- state.saldo ska ändras till att connecta till kredit istället-->
-              <figcaption class="l-text" id='top-logo-text'> {{ $t('credit') }}: {{ this.$store.state.creditLine }} {{ $t('org.tkn') }}</figcaption>
+              <figcaption class="l-text" id='top-logo-text'> {{ $t('credit') }}: {{ this.availableBalance(this.$store.state) }} {{ $t('org.tkn') }}</figcaption>
```

Comment:
Fine with replacing 'Available Credit' with 'Credit' (so long as it's also applied to My account -> Balance & limits - I understand this should happen automatically since most texts are now in en.json, but please double-check since the internationalisation is still not completely finished), as Eugene's work on the making the language consistent means that there wouldn't now be any ambiguity.

Reply:
done